manager/src/main/scala/org/broadinstitute/transporter/transfer/TransferListener.scala

```diff
@@ -106,6 +108,52 @@ class TransferListener private[transfer] (
     }
   }
 
+  /** Expand transfers that need to be expanded and update the processed transfers. */
+  private[transfer] def processUpdates(
+    statusUpdates: Vector[(TransferStatus, Json, UUID, UUID)],
+    now: Instant
+  ): IO[Int] = {
+    val transaction = for {
+      finalStatusUpdates <- statusUpdates.traverse {
+        case (TransferStatus.Expanded, info, transferId, requestId) =>
+          for {
+            priority <- List(
+              Fragment.const(s"select priority from $TransfersTable"),
+              fr"where id = $transferId"
+            ).combineAll.query[Short].unique
+            transfers <- info.as[List[Json]].liftTo[ConnectionIO]
+            transferRequests = transfers.map(TransferRequest(_, None))
+            transferIds <- controller.recordTransferRequest(
+              transferRequests,
+              requestId,
+              priority
+            )
+          } yield {
+            (
+              TransferStatus.Expanded: TransferStatus,
+              transferIds.asJson,
```

Comment:
I wonder if we should have this store an object, for consistency with the other cases / "self-documentation". Something like:
```json
{
  "expandedTo": ["id1", "id2"]
}
```
Thoughts?

Reply:
I will create a type for it, it will both be clearer and better for testing.